browser.js

```diff
@@ -59,6 +64,27 @@ queryHttps.cancel = () => {
 	xhr.abort();
 };
 
+const v6or4 = async options => {
+	let ip;
```

Comment:
This doesn't need to be defined at this scope.

Reply:
Sorry, but I don't understand what you meant

Reply:
It's not used outside the try catch, so it can be moved to line 75.